code/obj/storage/secure_closets.dm

```diff
@@ -868,6 +879,9 @@ ADMIN_INTERACT_PROCS(/obj/storage/secure/closet, proc/break_open)
 /obj/storage/secure/closet/civilian/ranch
 	name = "\improper Rancher supplies locker"
 	req_access = list(access_ranch)
+	icon_state = "secure_green"
+	icon_closed = "secure_green"
+	icon_opened = "secure_green-open"
```

Comment:
Not a part of this PR.

Reply:
Done + Clarion's pipe goes to mail loop for QM, just crosses over the crate transfer pipe so at a glance it looks like it goes into there